From 154cc73ca4d359621202399cc0c3c91058e56e79 Mon Sep 17 00:00:00 2001
From: Sam James <sam@cmpct.info>
Date: Wed, 22 Feb 2023 14:16:46 +0000
Subject: [PATCH] test: common: include <cstdlib> for abort() (fix build with
 GCC 13) (#1031)

GCC 13 (as usual for new compiler releases) shuffles around some
internal includes and so <cstdint> etc is no longer transitively included.

See https://www.gnu.org/software/gcc/gcc-13/porting_to.html.

Signed-off-by: Sam James <sam@gentoo.org>
---
 test/common/utils_assert.h | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/test/common/utils_assert.h b/test/common/utils_assert.h
index 1df8ae72ac..0123ab881e 100644
--- a/test/common/utils_assert.h
+++ b/test/common/utils_assert.h
@@ -1,5 +1,5 @@
 /*
-    Copyright (c) 2005-2022 Intel Corporation
+    Copyright (c) 2005-2023 Intel Corporation
 
     Licensed under the Apache License, Version 2.0 (the "License");
     you may not use this file except in compliance with the License.
@@ -20,6 +20,8 @@
 #include "config.h"
 #include "utils_report.h"
 
+#include <cstdlib>
+
 #define REPORT_FATAL_ERROR REPORT
 
 namespace utils {
